### PR TITLE
Adding jira

### PR DESCRIPTION
**Pull request by antgonza**
This is an initial test to see if we can add jira to the labadmin. Note that the current code is connecting to the test site of jira: https://jira.atlassian.com, as an anonymous user.

### CODE REVIEW

**Comment by wasade** [comment]
:+1:

**Comment by sjanssen2** [comment]
don't we have a test jira instance?

**Comment by josenavas** [comment]
I marked it as a WIP PR. We currently don't have a test Jira instance.

File: knimin/lib/ag_jira.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from jira import JIRA
+
+
+def jira_connect(server='https://jira.atlassian.com',
+                 user=None, password=None):
+    """Connect to JIRA server
+
+    Parameters
+    ----------
+    server: str, optional
```
Comment (sjanssen2) [inline knimin/lib/ag_jira.py]: why is server optional? Can it work without specifying a server? I would vote for making server mandatory

Comment (antgonza) [inline knimin/lib/ag_jira.py]: It's optional cause it has a default value, which is valid. BTW optional in this context means that the user doesn't have to pass it ...

File: .travis.yml [hunk 2]
```diff
@@ -6,12 +6,26 @@ services:
   - redis-server
 addons:
   postgresql: "9.3"
+  packages:
+    - oracle-java8-installer
```
Comment (wasade) [inline .travis.yml]: :(

File: knimin/lib/tests/test_ag_jira.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from unittest import TestCase, main
+
+from knimin.lib.ag_jira import jira_connect, get_projects
```
Comment (wasade) [inline knimin/lib/tests/test_ag_jira.py]: should this be `ag_jira` or just `jira`?

**Pull request merged**